src/olympia/signing/tests/test_views.py

```diff
@@ -94,11 +99,80 @@ def test_addon_does_not_exist(self, sign_version):
         assert addon.status == amo.STATUS_LITE
         sign_version.assert_called_with(addon.latest_version, is_beta=False)
 
+    @mock.patch('olympia.devhub.views.auto_sign_version')
+    def test_addon_does_not_exist_webextension(self, sign_version):
+        create_switch('addons-linter')
+
+        response = self.request(
+            'POST',
+            url=self.url(None, '1.0'),
+            addon='@create-webextension',
+            version='1.0')
+        assert response.status_code == 201
+
+        guid = response.data['guid']
+        addon = Addon.unfiltered.get(guid=guid)
+
+        assert addon.guid is not None
+        assert addon.guid != self.guid
+
+        version = Version.objects.get(addon__guid=guid, version='1.0')
+        assert version.files.all()[0].is_webextension is True
+        assert addon.has_author(self.user)
+        assert not addon.is_listed
+        assert addon.status == amo.STATUS_LITE
+        sign_version.assert_called_with(addon.latest_version, is_beta=False)
+
+    @mock.patch('olympia.devhub.views.auto_sign_version')
+    def test_optional_id_not_allowed_for_regular_addon(self, sign_version):
+        create_switch('addons-linter')
+        response = self.request(
+            'POST',
+            url=self.url(None, '1.0'),
+            addon='@create-version-no-id',
+            version='1.0')
```

Comment:
for tests like this that do not directly reference `version`, it's redundant to send it to `self.request()` because the default value should suffice. There are a couple other tests where I think you can also leave it off.